src/event.js

```diff
@@ -677,30 +683,24 @@ jQuery.Event.prototype = {
 	isImmediatePropagationStopped: returnFalse,
 
 	preventDefault: function() {
-		var e = this.originalEvent;
-
 		this.isDefaultPrevented = returnTrue;
 
-		if ( e && e.preventDefault ) {
-			e.preventDefault();
+		if ( this.originalEvent ) {
```

Comment:
This will break for uses like: `elem.trigger( $.Event( { type: "custom" } ) );`. In this case, the original event will just be `{ type: "custom" }`, so it won't contain a `preventDefault()` method.